### DISCUSSION TITLE
How to disable "X references" tooltip for scoped styles?

### DISCUSSION THREAD

**Discussion by Trinovantes** [vuejs/language-tools#1472] [Q&A] [answered]
Is there a setting that I can set to disable the "X references" tooltip for `<style scoped>`?

I personally don't find them very helpful and often doesn't work as expected

![Screenshot 2022-06-17 001514](https://user-images.githubusercontent.com/458691/174223653-a81a1a58-6198-46a8-b9a5-a94c578562bf.png)

On line 88 and 94, it generates the tooltip for the sass builtin function `math.div`

On line 124, it fails to find reference to my `<pre>` tag

**Comment by yaegassy** [comment, marked as answer]
Turn this setting off and try.

<img width="999" alt="volar-codelens-reference" src="https://user-images.githubusercontent.com/188642/174224352-f1d76d3e-4ac6-437e-ba94-0fc7bb5adfbc.png">

To change settings using `settings.json`, set `volar.codeLens.references` to `false`.

**settings.json**:

```jsonc
{
  "volar.codeLens.references": false,
}
```

**Reply by Trinovantes** [reply]
Thanks

**Reply by IlyaSemenov** [reply]
This was renamed to `vue.codeLens.enabled`.

**Comment by samueleiche** [comment]
for future seekers, it's now called `vue.codeLens.enabled`.
![image](https://user-images.githubusercontent.com/20043312/234816537-1f414f59-de2b-4c08-b4ac-bcbb6868de62.png)
